tsconfig.json

```diff
@@ -2,7 +2,9 @@
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
     "rootDir": "ts",
-    "outDir": "build"
+    "outDir": "build",
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true
```

Comment:
do these make sense to upstream to `gts`?

Reply:
I think they do. But they must be done together with the change that makes the generated `package.json` use typescript >=2.7.1.